src/ert/_c_wrappers/enkf/observations/summary_observation.py

```diff
@@ -12,8 +12,5 @@ class SummaryObservation:
     std: float
     std_scaling: float = 1.0
 
-    def updateStdScaling(self, factor: float, active_list: ActiveList) -> None:
```

Comment:
Remove this function while you are at it? The only call site outside of tests checks if `SummaryObservation`, so `semeio` could just do:

```python
sum_obs.std_scaling = new_val
```

Reply:
If not, can remove `active_list` from signature, it is never called with that outside of tests.